tabpy/tabpy_server/app/app.py

```diff
@@ -99,18 +138,31 @@ def run(self):
         settings = {}
         if self.settings[SettingsParameters.GzipEnabled] is True:
             settings["decompress_request"] = True
+
         application.listen(
             self.settings[SettingsParameters.Port],
             ssl_options=ssl_options,
             max_buffer_size=max_request_size,
             max_body_size=max_request_size,
             **settings,
-        )
+        ) 
 
         logger.info(
             "Web service listening on port "
             f"{str(self.settings[SettingsParameters.Port])}"
         )
+
+        if self.settings[SettingsParameters.ArrowEnabled]:
+            def start_pyarrow():
+                self.arrow_server = self._get_arrow_server(self.settings)
+                pa.start(self.arrow_server)
+
+            try:
+                _thread.start_new_thread(start_pyarrow, ())
+            except Exception as e:
+                print(e)
```

Comment:
Maybe replace these two print statements with `logger.critical(f"Unable to start pyarrow server: '{e}'")`